app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+  <div class="card-header">
+<h1>New Expense</h1>
+
+<%= form_with(model: @expense, url: expenses_path, local: true) do |form| %>
+  <% if @expense.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@expense.errors.count, "error") %> prohibited this expense from being saved:</h2>
+      <ul>
+        <% @expense.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+
+  <div class="field form-group">
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="field form-group">
+    <%= form.label :amount %>
+    <%= form.number_field :amount %>
+  </div>
+
+  <div class="field form-group">
+    <%= form.label :category_id %>
+    <%= form.collection_select :category_id, Category.all, :id, :name, prompt: true %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit "Create Expense" %>
+  </div>
+<% end %>
+<%= link_to "Back", expenses_path %>
+</div>
+</div>
```

Comment:
- [ ] Kindly improve the styling of the new expense form to match the design as demonstrated earlier.

  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/7ac7fb83-7866-47c6-a4bc-7fb321d33cd2)